### DISCUSSION TITLE
PatchTool 6.41 6/20

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#355] [Announcements]
6.41相容性更新；技改更新至6.41。一般漢化只到6.31

部分技能、狀態、對話因與 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/259 中提到的相同原因，目前會顯示原文。

## 直接用

新增漢化檔案，解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20230620.rar

## 其他檔案

檔案：

- FFXIVChnTextPatch-20230620.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。

選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20230620.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIVChnTextPatch-20230603.jar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.1.1.0/FFXIVChnTextPatch-20230603.jar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20230620.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

## 附註
根據 #354 提供的文本內容更新了漢化。除了：
* Action.csv: 31653

另，以下部分在刪除了我看不懂的指令碼後套用：
* InstanceContentTextData.csv: 35827, 35832

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.1.1.1'>PatchTool 6.41 6/20</a>.</em>

**Comment by Bahost8129** [comment]
請問我在使用FFXIVChnTextPatch-20230620.rar時出現
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/137670301/191e2226-ab51-490b-8801-58a0ede4e9f8)
該如何解決
跟目錄使用 C:\Users\z1020\Documents\My Games\FINAL FANTASY XIV - A Realm Reborn

**Reply by GpointChen** [reply]
你直接修改`FFXIVChnTextPatch\conf\global.properties`後打開程式看看？

**Reply by GpointChen** [reply]
另外，你可以先確認遊戲資料夾內是否有遊戲主程式，應該會位於 `FINAL FANTASY XIV - A Realm Reborn\game\ffxiv.exe`

**Reply by Bahost8129** [reply]
解決了，感謝您

**Comment by NightW15h** [comment]
不好意思 我使用steam玩ff14的 按了還原以後還是保持漢化了 請問有解決方法嗎 怕到時候更新會出問題

**Reply by GpointChen** [reply]
steam版應該可以直接右鍵>內容>檢查遊戲檔案完整性吧

**Reply by GpointChen** [reply]
或者是你可以用release中的backup檔案去覆蓋遊戲檔案，只是我不知道steam版檔案是否一樣
